### PR TITLE
1 deploy

### PR DESCRIPTION
**Pull request by kylem038**
@blakeworsley Hey man - I changed the jQuery require to use Steve's approach. 

With regards to the key bind for toggling the color dropper - I noticed with the code you had set up it only triggered only upon pressing ESC & not on E (aka keycode 69). I spent a good amount of time trying to implement the 'cmd+D' event for triggering the colordropper but apparently listening for 2 key events simultaneously in jQuery isn't as easy as it seems. What do you think about just keeping the key bind at just D?

### CODE REVIEW

**Comment by blakeworsley** [comment]
I'm totally cool to keep the bind on the D for now. I will look into getting the command D working properly. I think we have to do it through the main window as opposed to adding an event listener.

File: build/config.gypi [hunk 1]
```diff
@@ -0,0 +1,67 @@
+# Do not edit. File was generated by node-gyp's "configure" step
```
Comment (blakeworsley) [inline build/config.gypi]: Just out of curiosity, is this file necessary?

**Review by blakeworsley — CHANGES_REQUESTED** [review]

File: build/config.gypi (hunk 1 above)
Comment (kylem038) [reply build/config.gypi]: In a word, no. It's a file that helped me get past the problem I was having when I couldn't run the app on my computer. I added it to my gitignore but only after my first push, so even if I push again it still shows up. Isn't there a way I can remove it in the PR?

**Comment by kylem038** [comment]
Well that wasn't as hard as I thought. Got rid of that config file.

**Comment by blakeworsley** [comment]
Awesome! Thanks Kyle!
